src/actions/actions.ts

```diff
@@ -1652,6 +1652,114 @@ class MoveLineBegin extends BaseMovement {
 }
 
 @RegisterAction
+class MoveScreenLineBegin extends BaseMovement {
```

Comment:
In accordance with #519 (sorry for referring to an issue that was clearly made after your PR!), could you make this into an abstract base class?